lm_eval/tasks/haerae.py

```diff
@@ -0,0 +1,56 @@
+from lm_eval.base import MultipleChoiceTask
```

Comment:
Is there a recommended reference/citation for this new benchmark?

Reply:
our paper is still under review but we will add it as soon as it is free to be public. in the meantime we do have a short blog post (https://github.com/EleutherAI/hae-rae/tree/main/HAE-RAE%20Bench) will this help?